config.js

```diff
@@ -492,10 +416,12 @@ function testDroppedArgv(argv) {
     console.log("\n");
 }
 
-function checkUnsupportedOgspvAI(ogspv, ogsPvAIs) {
-    const upperCaseAIs = ogsPvAIs.map(e => e.toUpperCase());
+function testUnsupportedOgspvAI(ogspv, ogsPvAIs) {
```

Comment:
possibly call this `validateUnsupportedOgspvAI`?

Reply:
yes its fine